WireAPI/Tests/WireAPITests/APIs/ConversationsAPI/ConversationsAPITests.swift

```diff
@@ -527,7 +527,7 @@ final class ConversationsAPITests: XCTestCase {
             for sut in suts {
                 taskGroup.addTask {
                     // Then
-                    await self.XCTAssertThrowsError(ConversationsAPIError.unsupportedEndpointForAPIVersion) {
+                    await self.XCTAssertThrowsErrorAsync(ConversationsAPIError.unsupportedEndpointForAPIVersion) {
```

Comment:
The complier was having a hard time distinguishing this async variant (we define it WireTestingPackage) and the existing non-async variant. Hence, I just gave it a different name.